crates/astria-composer/src/executor/builder.rs

```diff
@@ -15,7 +15,7 @@ use astria_core::{
 use astria_eyre::eyre::{
     self,
     eyre,
-    WrapErr as _,
```

Comment:
unclear as to why this change is here to me

Reply:
Should be resolved with be387af.

Reply:
Undo the change, use `WrapErr as _`